### PR TITLE
Email help updates

### PR DESCRIPTION
**Pull request by jameshupp**
## Changes
-added FAQs re email: conversation settings, filtering, and replying with comments

-small edits to other files

## Review 
@phirefly - make sure I’m accurately addressing the needs here

## Comments
This precedes an update email to users. I’ll need to add in the URLs to that once this is in production

### CODE REVIEW

File: app/views/help/email_conversations.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<% title "Why can’t I find emails about a purchase request I’m involved in?"%>
+
+Gmail can group emails into conversations based on the subject line. If you use this function, all your emails about a single purchase request will be grouped together, with only the most recent one visible in the inbox. If you see a number in parentheses after the list of senders like `(4)`, that is how many messages are in that conversation. Open that email to see all of the earlier ones.
```
Comment (phirefly) [inline app/views/help/email_conversations.md]: Can we indicate that the default setting of 'on' for conversation view is the recommended setup?

**Comment by phirefly** [comment]
Added one comment. Otherwise, :+1:

File: app/views/help/email_conversations.md (hunk 1 above)
Comment (jameshupp) [inline app/views/help/email_conversations.md]: Good note. New edit addresses it.

**Comment by pkarman** [comment]
@phirefly feel free to merge if this looks ok to you.

**Comment by phirefly** [comment]
:ship: it

**Pull request merged**